packages/styles/table.css

```diff
@@ -30,6 +36,44 @@
   border-bottom: 2px solid var(--table-row-border-color);
 }
 
+.TableHeader .sort-button {
+  background-color: transparent;
+  width: 100%;
+  /* a fixed height prevent TableHeader from being jumpy */
+  height: 1.625rem;
+  display: flex;
+  align-items: center;
+  align-content: center;
+  padding-left: 0;
+  font-family: 'Roboto', Helvetica, Arial, sans-serif;
```

Comment:
Why is `font-family` being set here, shouldn't that be inherited?

Reply:
I know, somehow the button does not inherit the font-family here. It would not allow me to change font-weight until font-family is set here.